test/src/unit-capi-dense_array.cc

```diff
@@ -676,34 +676,31 @@ void DenseArrayFx::write_dense_array_by_tiles(
     const int64_t domain_size_1,
     const int64_t tile_extent_0,
     const int64_t tile_extent_1) {
-  // Error code
-  int rc;
-
   // Other initializations
-  auto buffer = generate_2D_buffer(domain_size_0, domain_size_1);
-  int64_t cell_num_in_tile = tile_extent_0 * tile_extent_1;
-  auto buffer_a1 = new int[cell_num_in_tile];
+  auto buffer{generate_2D_buffer(domain_size_0, domain_size_1)};
+  int64_t cell_num_in_tile{tile_extent_0 * tile_extent_1};
+  auto buffer_a1{new int[cell_num_in_tile]};
   for (int64_t i = 0; i < cell_num_in_tile; ++i)
     buffer_a1[i] = 0;
-  void* buffers[2];
-  buffers[0] = buffer_a1;
-  uint64_t buffer_sizes[2];
-  int64_t index = 0L;
-  uint64_t buffer_size = 0L;
+  void* buffers[2]{buffer_a1, nullptr};
+
+  uint64_t buffer_sizes[2]{0UL, 0UL};
```

Comment:
Shouldn't the first be `cell_num_in_tile`?

Reply:
Not sure.  The original code (dev) sets it much later in the function (line 750 in dev, line 747 in branch).